### DISCUSSION TITLE
GitHub actions queued and won't deploy or cancel

### DISCUSSION THREAD

**Discussion by smakdonald** [community/community#51762] [Actions]
### Select Topic Area

Question

### Body

All of the actions I attempt to deploy end up queued and refuse to deploy. It seems this has been a recurring issue for others in the past but no consistent solution has been found. When I attempt to cancel the workflow GitHub says "Failed to cancel workflow." The actions have been queued for almost five days now with no changes. 

<img width="1049" alt="Screenshot 2023-04-02 at 11 21 31 AM" src="https://user-images.githubusercontent.com/107808103/229365638-6faaed42-5aab-4038-921e-2e07c23e977c.png">

<img width="1406" alt="Screenshot 2023-04-02 at 11 22 16 AM" src="https://user-images.githubusercontent.com/107808103/229365678-d1ab0eef-c9ad-4328-ad6f-3687e3c5e455.png">

**Comment by Akash1134** [comment]
Hi @smakdonald , Sorry that you're facing issues using Actions. Does it help if you manually set the stuck reruns to completed/failure?

**Comment by smakdonald** [comment]
Hi Akash,

No, I don't think it does anything.
